wget/wget.sh

```diff
@@ -27,13 +27,14 @@ downloadimages() {
     mkdir .imagecache
     cd .imagecache
     fakebrowser "$1"
-    DOMAIN2=$(echo "$1" | grep -Eo 'http[s:]{,3}//[^/]*/')
+    DOMAIN2=$(grep -Eo 'http[s:]{,3}//[^/]*/' <<< "$1")
     DOMAIN="${DOMAIN2%/}"
     SINDEX="$(ls)"
-    downloadformat "jpg" "$SINDEX" "$DOMAIN"
-    downloadformat "png" "$SINDEX" "$DOMAIN"
-    downloadformat "jpeg" "$SINDEX" "$DOMAIN"
-    downloadformat "gif" "$SINDEX" "$DOMAIN"
+
+    for FMT in jpg png jpeg gif; {
+        downloadformat "$FTM" "$SINDEX" "$DOMAIN"
+    }
```

Comment:
To avoid code repetition, and make your life a lil easier if you need to add or remove possible extensions.

Reply:
It works (after fixing the typo), but my parser (shfmt) doesn't like this one. Should I report that or is that intentional by shfmt?

Reply:
Lol Sorry about that typo. Strange that it doesn't work. Maybe just leave it as it was then. Might be worth reporting that, because that seems odd, unless I'm missing completely something. I've never heard of shfmt.